### PR TITLE
sct: enable to copy user repos instead of just orgs

### PR DESCRIPTION
**Pull request by jhchabran**
This enables to copy from repos from a user instead of just orgs. This is needed so we can copy the `testing/*` repos for Batch changes. 

## Test plan

Manually tested.

### CODE REVIEW

File: dev/scaletesting/codehostcopy/github.go [hunk 1]
```diff
@@ -37,21 +39,41 @@ func NewGithubCodeHost(ctx context.Context, def *CodeHostDefinition) (*GithubCod
...
-		ListOptions: github.ListOptions{},
-	}
 	var repos []*github.Repository
-	for {
-		rs, resp, err := g.c.Repositories.ListByOrg(ctx, g.def.Path, &opts)
-		if err != nil {
-			return nil, err
+
+	if strings.HasPrefix(g.def.Path, "@") {
+		// If we're given a user and not an organization, query the use repos.
```
Comment (burmudar) [inline dev/scaletesting/codehostcopy/github.go:45]:
```suggestion
		// If we're given a user and not an organization, query the user repos.
```

**Review by burmudar — APPROVED** [review]

File: dev/scaletesting/codehostcopy/github.go [hunk 2]
```diff
@@ -37,21 +38,41 @@ func NewGithubCodeHost(ctx context.Context, def *CodeHostDefinition) (*GithubCod
...
-	}
 	var repos []*github.Repository
-	for {
-		rs, resp, err := g.c.Repositories.ListByOrg(ctx, g.def.Path, &opts)
-		if err != nil {
-			return nil, err
+
+	if strings.HasPrefix(g.def.Path, "@") {
+		// If we're given a user and not an organization, query the user repos.
+		opts := &github.RepositoryListOptions{
```
Comment (burmudar) [inline dev/scaletesting/codehostcopy/github.go:45]:
```suggestion
		opts := github.RepositoryListOptions{
```
Minor but for consistency

File: dev/scaletesting/codehostcopy/github.go [hunk 3]
```diff
@@ -37,21 +38,41 @@ func NewGithubCodeHost(ctx context.Context, def *CodeHostDefinition) (*GithubCod
...
+			ListOptions: github.ListOptions{},
+		}
+		for {
+			rs, resp, err := g.c.Repositories.List(ctx, strings.Replace(g.def.Path, "@", "", 1), opts)
```
Comment (burmudar) [inline dev/scaletesting/codehostcopy/github.go:49]:
```suggestion
			rs, resp, err := g.c.Repositories.List(ctx, strings.Replace(g.def.Path, "@", "", 1), &opts)
```
If the previous change is accepted 👯

Comment (jhchabran) [reply dev/scaletesting/codehostcopy/github.go]: Yes of course!

**Pull request merged**